test/langtools/tools/javac/T8314423.java

```diff
@@ -2,8 +2,7 @@
  * @test  /nodynamiccopyright/
  * @bug 8314423
  * @summary Multiple patterns without unnamed variables
- * @compile/fail/ref=T8314423.out -XDrawDiagnostics T8314423.java
- * @compile --enable-preview --source ${jdk.version} T8314423.java
+ * @compile T8314423.java
```

Comment:
I think I would keep the existing `@compile` as well, with `--release 21`, to verify the source level checks are working.